Dockerfile

```
@@ -17,6 +17,10 @@ FROM restore AS test
 ENV CI=true
 RUN yarn test
 
+FROM test AS coverage
+ENV CI=true
```

La variable de entorno no es necesaria porque ya está definido en la etapa de `test`

```suggestion
FROM test AS coverage
```

---

Ok. Changed.